tests/gk-deploy/test_gk_deploy_basic.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(cd $(dirname ${0}); pwd)
+STUBS_DIR="${SCRIPT_DIR}/stubs"
+TESTS_DIR="${SCRIPT_DIR}/.."
+INC_DIR="${TESTS_DIR}/common"
+BASE_DIR="${SCRIPT_DIR}/../.."
+DEPLOY_DIR="${BASE_DIR}/deploy"
+
+GK_DEPLOY="${DEPLOY_DIR}/gk-deploy"
+TOPOLOGY="${DEPLOY_DIR}/topology.json.sample"
+
+PATH="${STUBS_DIR}:$PATH"
+
+source "${INC_DIR}/subunit.sh"
+
+
+
+test_missing_topology () {
+	${GK_DEPLOY} -y
+	if [ "x$?" = "x0" ]; then
+		echo "ERROR: gk_deploy without toplogoy succeeded"
+		return 1
+	fi
+
+	return 0
+}
+
+test_cli_not_found () {
+	local saved_path="${PATH}"
+	PATH="/usr/bin:/bin"
```

Comment:
That's right. Going to use a different path that won't work, maybe even does not exist. Like "/foo".